docs/custom-operator-image.md

```diff
@@ -0,0 +1,22 @@
+# Customer Operator container image
```

Comment:
```suggestion
# Customer Operator container images
```

Reply:
Just double checking - do you mean "Customer Operator container images" here or "Custom Operator container images"?